test-load/pkg/exec/helpers.go

```diff
@@ -0,0 +1,68 @@
+package exec
+
+import (
+	"bufio"
+	"bytes"
+	"os"
+	"strings"
+
+	"github.com/alessio/shellescape"
+)
+
+func PrettyCommand(name string, args ...string) string {
+	var out strings.Builder
+	out.WriteString(shellescape.Quote(name))
+	for _, arg := range args {
+		out.WriteByte(' ')
+		out.WriteString(shellescape.Quote(arg))
+	}
+	return out.String()
+}
+
+func RunErrorForError(err error) *RunError {
+	var runError *RunError
+	for {
+		if rErr, ok := err.(*RunError); ok {
+			runError = rErr
+		} else {
+			break
+		}
+	}
+	return runError
+}
+
+func CombinedOutputLines(cmd Cmd) (lines []string, err error) {
```

Comment:
The golang library https://pkg.go.dev/os/exec#Cmd.CombinedOutput looks like does exactly like this, why we need to build a library?